packages/block-library/src/navigation/edit/index.js

```diff
@@ -559,6 +560,23 @@ function Navigation( {
 		{ open: overlayMenuPreview }
 	);
 
+	useEffect( () => {
+		if ( isFallback ) {
+			__unstableMarkNextChangeAsNotPersistent();
+			replaceInnerBlocks( clientId, [ createBlock( 'core/page-list' ) ] );
+		}
+	}, [ isFallback ] );
+
+	const innerBlocksProps = useInnerBlocksProps( blockProps );
```

Comment:
Is there any reason for not directly setting `'core/page-list'` as an inner blocks template?

Reply:
It doesn't really work because we need to not register a save and we will always look to see if the innerBlocks have been there from the markup or if they're added programatically as a fallback. UnsavedInnerBlocks will create an undo when adding the template.

Reply:
Thanks for checking, @draganescu.